PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -207,6 +207,11 @@ public function close($reply_code = 0, $reply_text = '', $method_sig = array(0,
         ), false, $this->channel_rpc_timeout);
     }
 
+    public function markClosed()
```

Comment:
There is no point of adding public wrapper for protected function. Also we don't want to have potentially harmful public method.
We should either make public `do_close()` or do something smarter. I suggest this way:
```
public function closeIfDisconnected(): void
{
    if ($this->connection && !$this->connection->isConnected()) {
        $this->do_close();
    }
}
```

Reply:
Agree, I'll change this